.github/pull_request_template.md

```diff
@@ -40,7 +41,8 @@
   - [ ] Includes bug reproducer
   - [ ] No tests
 - Teuthology
-  - [ ] Completed teuthology run
+  - [ ] Teuthology testing needed
+  - [ ] Completed teuthology testing
   - [ ] No teuthology test necessary (e.g., documentation)
```

Comment:
I think the question here is if we want to "needs-qa" option... this is similar to what the existing tag alread offers.  and it means once you check the box it will appear green on github.  not sure if that is a good or bad thing, TBH... since i also scan those statuses in the pull request list to see which pull requests are ready to test or review.

Reply:
I see your point, just that I saw people selecting completed teuthology testing while opening new PR ;-) 
Amended it for do we need-qa for this PR.